lib/generators/hyrax/work/templates/locale.es.yml.erb

```diff
@@ -0,0 +1,37 @@
+en:
+  hyrax:
+    icons:
+      <%= file_name %>:     'fa fa-file-text-o'
+    select_type:
+      <%= file_name %>:
+        # TODO: translate `human_name` into Spanish
+        name:               "<%= human_name.titleize %>"
+        # TODO: translate `human_name` into Spanish
+        description:        "<%= human_name %> trabajos"
+  simple_form:
+    labels:
+      <%= file_name %>:
```

Comment:
@jcoyne I'm not sure. We did this for a reason, I assume: https://github.com/projecthydra/sufia/commit/f6decc18996b78c3a59c9371b89761cccb6614b8